Add "The Roundtable" news and wiki articles (#7616)
ppy/osu-wiki: news/2022-07-08-the-roundtable.md

---
layout: post
title: "The Roundtable"
date: 2022-07-08 10:00:00 +0000
---

16 osu! warriors battle in the inaugural edition of **The Roundtable**, osu!'s first major in-person invitational tournament!

![](/wiki/shared/news/2022-07-08-the-roundtable/roundtable_banner.png)

> While LAN events are nothing new to the osu! scene, [**The Roundtable**](/wiki/Tournaments/The_Roundtable) takes a different approach.

Reviewer: I'd honestly reword this sentence to something like

```suggestion
While LAN events are nothing new to the osu! scene, [**The Roundtable**](/wiki/Tournaments/The_Roundtable) is pushing the boundaries with a very stacked roster of players!
```

This **3 day** event from July 8-10 features **3 different tournaments**, all streamed from [BTMC](https://osu.ppy.sh/users/3171691)'s [Twitch channel](https://twitch.tv/BTMC) with comfy couch commentary and personal interactions with everyone involved!

## Schedule

| Event | Timestamp |
| --: | :-- |
| [16 player battle royale](/wiki/Tournaments/The_Roundtable/16_player_battle_royale) | 2022-07-06 (21:00 UTC +0) |
| [2v2 team tandem tournament](/wiki/Tournaments/The_Roundtable/2v2_team_tandem_tournament) | 2022-07-07 (18:00 UTC +0) |
| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) | 2022-07-07 (22:00 UTC +0) |

> | [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) (top 8) | 2022-07-08 (17:30 UTC +0) |

Reviewer: Are the dates wrong here? Above it says july 8-10 but this is 6-8, and mostly in the past..

Reviewer 2: Indeed, they should be 8-10th July

Author: 8-10 is right yeah

> | [16 player battle royale](/wiki/Tournaments/The_Roundtable/16_player_battle_royale) | 2022-07-08 (21:00 UTC +0) |
> | [2v2 team tandem tournament](/wiki/Tournaments/The_Roundtable/2v2_team_tandem_tournament) | 2022-07-09 (18:00 UTC +0) |
> | [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) | 2022-07-09 (22:00 UTC +0) |
> | [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) (top 8) | 2022-07-10 (17:30 UTC +0) |

Reviewer 3: ```suggestion
| [16 player battle royale](/wiki/Tournaments/The_Roundtable/16_player_battle_royale) | 2022-07-08 (21:00 UTC) |
| [2v2 team tandem tournament](/wiki/Tournaments/The_Roundtable/2v2_team_tandem_tournament) | 2022-07-09 (18:00 UTC) |
| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) | 2022-07-09 (22:00 UTC) |
| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) (top 8) | 2022-07-10 (17:30 UTC) |
```
also if it's important for viewers to get the right time, something like <https://github.com/ppy/osu-wiki/pull/7582/files#diff-b0dd85f0e03a4c42664ec75774aa150ed00925a33b09a06c0efe7bbf1b4b5693R106> might be helpful, they've been adding this to OWC schedules recently

Reviewer: I think adding a link to show comparisons or local timezones is good, but that page used in the linked article is... useless? It just shows the UTC time for me, with ads plastered everywhere. Best to find an alternative site which actually shows the user's local timezone automatically.

Would be cool to add support for this at a wiki level at some point.

### 16 player battle royale

Kicking off **The Roundtable** is a [**16 player battle royale**](/wiki/Tournaments/The_Roundtable/16_player_battle_royale)!

The LAN setup has 8 computers, so the players will be randomized into two groups of 8. Each group will play **7** maps in a multiplayer lobby, and the lowest scorer of each map will be eliminated.

Once both groups have been narrowed down to a single player, the finalists will face head to head in a **best of 13 competition!**

### 2v2 team tandem tournament

The event's second day launches with an unorthodox [**2v2 team tandem tournament**](/wiki/Tournaments/The_Roundtable/2v2_team_tandem_tournament).

Honestly speaking, this is more about having fun than competition. "Tandem" in this context means there will be **two players clicking circles on one setup** — one aiming and the other tapping!

> Players will form four groups of four and face off in a single elimination tournament -- until the last group remains as the victor.

Reviewer 3: ```suggestion
Players will form four groups of four and face off in a single elimination tournament — until the last group remains as the victor.
```
fun with dashes!

> ### 1v1 double elminiation tournament

Reviewer 4: ```suggestion
### 1v1 double elimination tournament
```

The main event of **The Roundtable** is a high stakes [**1v1 double elimination tournament!**](/wiki/Tournaments/The_Roundtable/1v1_double_elimination).

This tournament will last two days: The first day of matches will determine the Top 8 players, while the second day is reserved for illustrating the talents of the best players at **The Roundtable.**

> This classic 1v1 has the largest set of prizes, so expect to see players perform - in person - at the highest caliber.

Reviewer 3: ```suggestion
This classic 1v1 has the largest set of prizes, so expect to see players perform — in person — at the highest caliber.
```

## Players

So who's coming to this thing anyways?

**15** of North America's strongest competitors have been invited, including former and current [OWC](/wiki/Tournaments/OWC) champions.

> Take a peak at everything at the full list below, including a special teaser reveal for osu!'s rank 1 player:

Reviewer 2: ```suggestion
Take a peek at everything at the full list below, including a special teaser reveal for osu!'s rank 1 player:
```

> Take a peak at everything at the full list below, including a special teaser reveal for osu!'s rank 1 player:

Reviewer 5: Take a peek at everything at the full list below, including a special teaser reveal for osu!'s rank 1 player:
